db/seeds.rb

```diff
@@ -6,9 +6,11 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-category1 = Category.create(name: 'Coffee')
-category2 = Category.create(name: 'Brewing')
-category3 = Category.create(name: 'Merchandise')
+category1 = Category.create(name: 'Coffee', image: "fa fa-coffee")
+category2 = Category.create(name: 'Brewing', image: "fa fa-tint")
+category3 = Category.create(name: 'Merchandise', image: "fa fa-gratipay")
+category4 = Category.create(name: 'Chow', image: "fa fa-cutlery")
```

Comment:
Useless assignment to variable - `category4`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.